### PR TITLE
Improvement: managing-labels page. Added link for page explaining how to use emoji

### PR DESCRIPTION
**Pull request by AllineMi**
The linked page also has a link for the Emoji-Cheat-Sheet, which can be very helpful.

### Why:

The page [managing labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels), through one of the images, shows that it is possible to add emojis to the label name, but it does not explain exactly how to do it.
The page also does not show what emojis are available.

### What's being changed:
Added a single item to the section _Further reading_ with a link for the [using-emoji](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji) section of the [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) page.
The linked page explains how to use emojis as well as it contains a link for a sheet with all the emojis available.

### Screenshot
Below is link of a screenshot showing the result with the suggested change implemented.
[https://imgur.com/a/Cjbqug6](https://imgur.com/a/Cjbqug6)

### Check off the following:

- [X] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@AllineMi 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/issues/using-labels-and-milestones-to-track-work/managing-labels.md [hunk 1]
```diff
@@ -107,4 +107,5 @@ Deleting a label will remove the label from issues and pull requests.
 ## Further reading
 - "[Filtering and searching issues and pull requests](/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests)"{% ifversion fpt or ghes or ghec %}
 - "[Managing default labels for repositories in your organization](/articles/managing-default-labels-for-repositories-in-your-organization)"{% endif %}{% ifversion fpt or ghec %}
-- "[Encouraging helpful contributions to your project with labels](/communities/setting-up-your-project-for-healthy-contributions/encouraging-helpful-contributions-to-your-project-with-labels)"{% endif %}
+- "[Encouraging helpful contributions to your project with labels](/communities/setting-up-your-project-for-healthy-contributions/encouraging-helpful-contributions-to-your-project-with-labels)"{% endif %}{% ifversion fpt or ghec %}
+- "[Basic writing and formatting syntax: using emoji](/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji)"{% endif %}
```
Comment (mchammer01) [inline content/issues/using-labels-and-milestones-to-track-work/managing-labels.md:111]: This change LGTM but we use article names in hyperlinked text, even when linking to an anchor inside articles
```suggestion
- "[Basic writing and formatting syntax](/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji)"{% endif %}
```

**Review by mchammer01 — DISMISSED** [review]
@AllineMi - thanks for your contribution ✨ 
I've made a minor suggestion relating to our content model, that I'm going to apply.
I'll merge this as soon as I can 🙂

**Review by mchammer01 — DISMISSED** [review]
Re-approving 🤦🏻

**Review by mchammer01 — COMMENTED** [review]
I noticed that https://docs-16606-a94447.preview.ghdocs.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax is available to all GitHub products, so I'm removing the versioning as it's not necessary.

**Review by mchammer01 — APPROVED** [review]
LGTM ✨ 
(trying to trigger an updated deployment)

**Comment by mchammer01** [comment]
I can't see an updated deployment with my latest changes, but as all the checks are passing, I'm going to merge this PR.

**Pull request merged**